apps/ckit-app/src/hooks/useSendIssueTx.tsx

```diff
@@ -0,0 +1,68 @@
+import { Address, HexNumber, Hash } from '@ckb-lumos/base';
+import { Modal } from 'antd';
+import { MintSudtBuilder, RecipientOptions } from 'ckit';
+import React from 'react';
+import { useMutation, UseMutationResult } from 'react-query';
+import { useConfigStorage } from './useConfigStorage';
+import { CkitProviderContainer, WalletContainer } from 'containers';
+import { hasProp } from 'utils';
+
+export interface SendIssueTxInput {
+  recipient: Address;
+  amount: HexNumber;
+  operationKind: 'invite' | 'issue';
+  setModalVisible: (visible: boolean) => void;
```

Comment:
an UI business should NOT in a hooks, this leads to coupling of UI and business logic
```suggestion
```

we can just

```ts
const {mutate: sendTx} = useSendIssueTx()

onSubmit: () => {
  return sendTx().then(
    () => // do something on success,
    () => do something on error,
  )
}
```

Reply:
fixed